### PR TITLE
Update gltf-model to glTF 2.0.

### PR DESCRIPTION
**Pull request by donmccurdy**
Example assets bumped with https://github.com/aframevr/assets/pull/27.

the glTF2.0 model is not backward-compatible, so users upgrading to A-Frame 0.7.0 but wanting to continue using their glTF1.0 assets will need to use [`gltf-model-legacy` frome aframe-extras](https://github.com/donmccurdy/aframe-extras/tree/master/src/loaders).

Fixes https://github.com/aframevr/aframe/issues/2693.

### CODE REVIEW

**Comment by donmccurdy** [comment]
Hm, in the interest of not breaking 0.6.0 examples, should the assets be added to a new folder, leaving the glTF 1.0 ones alone?

**Comment by dmarcos** [comment]
I think we won't break anything because the site won't deploy until `0.7.0`. Am I right?

File: docs/components/gltf-model.md [hunk 1]
```diff
@@ -91,17 +96,18 @@ rendering.
...
-- [glTF Workflow for A Saturday Night](https://blog.mozvr.com/a-saturday-night-gltf-workflow/)
-- [FBX &rarr; glTF][fbx-converter] - Coming soon.
-- [COLLADA &rarr; glTF][collada-converter]
+Over 100,000 glTF models are free for download on [Sketchfab][sketchfab], and
+various exporters and converters converters are available:
+
+- [Blender Exporter](blender-exporter)
 - [OBJ &rarr; glTF][obj-converter]
+- [COLLADA &rarr; glTF][collada-converter]
+- [glTF Workflow for A Saturday Night](https://blog.mozvr.com/a-saturday-night-gltf-workflow/)
```
Comment (dmarcos) [inline docs/components/gltf-model.md]: Is the blog post still relevant? Is it worth to bring it up to date?

**Comment by dmarcos** [comment]
It looks good to me. I think merging this won't affect the current deployed examples on the site. I was wondering if the `a-saturday-night` blog post should be updated with a glTF2 workflow (https://blog.mozvr.com/a-saturday-night-gltf-workflow/) but we should first update the assets to glTF2. Maybe not worth now. Should we keep the link then? Is it the info in the blog post still relevant?

**Comment by dmarcos** [comment]
Thank you very much! This makes me ultra happy. 😄 🎉 🎈 🎆 🍾

**Comment by donmccurdy** [comment]
@dmarcos Yay! Regarding the `a-saturday-night` blog post, a lot of what's written there is explaining how to work around limitations in the old Blender exporter, by exporting to COLLADA first. The new Blender exporter is much more reliable, and COLLDA2GLTF's glTF2.0 support is not finished, so I think we want to recommend using the Blender Exporter these days.

My vote would be to write a shorter new blog post, and then put an `UPDATE: ...` at the top of the old one linking to that. Unfortunately it's unlikely I would have time to write that in the next couple weeks. 😕

**Pull request merged**